### DISCUSSION TITLE
It's not working on Ethereum?

### DISCUSSION THREAD

**Discussion by ndtrung00** [fuzzland/ityfuzz#360] [Q&A] [answered]
Is there a reason, why the known-working hacks on the docs are only the one from BSC? https://docs.ityfuzz.rs/tutorials/exp-known-working-hacks

I have indeed also tried to fuzz on Ethereum myself and it always yield something like: 
`Error: error sending request for url (https://eth.llamarpc.com/): operation timed out`

any tips for this case?

**Comment by shouc** [comment, marked as answer]
Seems to be RPC issues. You can use your own RPC by setting 'ETH_RPC_URL=https://xxxx' environment variable. If you want to run on old block, make sure the node of RPC is running in Archive mode.

**Comment by ndtrung00** [comment]
thats righttt. works again! Thank you so much!
Wondering why it doesn't happen when you fuzz on BSC
